janusgraph-dist/src/assembly/static/bin/janusgraph.sh

```diff
@@ -83,7 +83,7 @@ wait_for_cassandra() {
     while [ $now_s -le $stop_s ]; do
         echo -n .
         # The \r\n deletion bit is necessary for Cygwin compatibility
-        statusbinary="`$BIN/../cassandra/bin/nodetool statusbinary 2>/dev/null | tr -d '\n\r'`"
+        statusbinary="`$BIN/../cassandra/bin/nodetool -Dcom.sun.jndi.rmiURLParsing=legacy statusbinary 2>/dev/null | tr -d '\n\r'`"
```

Comment:
This disables new validations of URL parsing which seems to have been added for security reasons. Is it really necessary to disable this?

Reply:
It fixes the test.